src/frontend/components/LiveVideoResizer/index.spec.tsx

```diff
@@ -0,0 +1,108 @@
+import { act, render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import React from 'react';
+
+import { useLivePanelState } from 'data/stores/useLivePanelState/index';
+import { LiveVideoResizer } from '.';
+
+const LeftCompo = <p>left component</p>;
+const RightCompo = <p>right component</p>;
+
+const mockedSetSavedPanelWidthPx = jest.fn();
+
+describe('<LiveVideoResizer />', () => {
+  beforeAll(() => {
+    jest.resetAllMocks();
+  });
+
+  it('renders component with panel opened and then clicks on close button', async () => {
+    useLivePanelState.setState({
+      isPanelVisible: true,
+    });
+    render(
+      <LiveVideoResizer
+        isReadyToDisplayRightElement={true}
+        isPanelOpen={true}
+        leftElement={LeftCompo}
+        rightElement={RightCompo}
+        savedPanelWidthPx={300}
+        setSavedPanelWidthPx={mockedSetSavedPanelWidthPx}
+      />,
+    );
+    screen.getByText('left component');
+    screen.getByText('right component');
+
+    const closeButton = screen.getByRole('button');
+    act(() => userEvent.click(closeButton));
+
+    expect(useLivePanelState.getState().isPanelVisible).toEqual(false);
+    expect(mockedSetSavedPanelWidthPx).toHaveBeenCalledTimes(1);
```

Comment:
You should probably add a test showing that the `right component` is not available anymore ?